org.d3cod3.Mosaic.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be upstreamed if possible

Reply:
And it will, just as the desktop file. Mosaic has a lot of moving parts, and for now I want to see if I can get it working as a flatpak (not sure it will tbh). I'll address upstream issues after that